nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -24,7 +24,7 @@
  * 3. The current feature behavior has been tested with MNIST.
  * 4. mnist.json has 'gst_caps' containing the information below.
  * "gst_caps":"other/tensors, format=(string)static, framerate=(fraction)0/1,
- *  num_tensors=(int)2, dimensions=(string)1:1:784:1,1:1:10:1,
+ *  num_tensors=(int)2, dimensions=(string)1:1:784:1.1:1:10:1,
```

Comment:
typo?

Reply:
Thank you but multi-dimensions are identified as "." instead of "," in caps.